Tests/ArcGISToolkitTests/PreplannedMapModelTests.swift

```diff
@@ -279,18 +279,61 @@ class PreplannedMapModelTests: XCTestCase {
         let area = try XCTUnwrap(areas.first)
         let areaID = try XCTUnwrap(area.portalItem.id)
         
+        let model = PreplannedMapModel(
+            offlineMapTask: task,
+            mapArea: area,
+            portalItemID: portalItem.id!,
+            preplannedMapAreaID: areaID,
+            // User notifications in unit tests are not supported, must pass false here
+            // or the test process will crash.
+            showsUserNotificationOnCompletion: false
+        )
+        
         defer {
             // Clean up JobManager.
             OfflineManager.shared.jobManager.jobs.removeAll()
             
             // Clean up folder.
-            let directory = FileManager.default.preplannedDirectory(
-                forPortalItemID: portalItem.id!,
-                preplannedMapAreaID: areaID
-            )
-            try? FileManager.default.removeItem(at: directory)
+            model.removeDownloadedPreplannedMapArea()
         }
         
+        var statuses: [PreplannedMapModel.Status] = []
+        var subscriptions = Set<AnyCancellable>()
```

Comment:
Not a big deal, but I suggest using syntactic sugar for collection initialization.

```suggestion
        var subscriptions: Set<AnyCancellable> = []
```